MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -431,17 +433,25 @@
 
 		mvkEnableFlags(mtlUsage, MTLTextureUsageRenderTarget);
 	}
-
+	
 	// Create view on, but only on color formats, or combined depth-stencil formats if supported by the GPU...
-	if (mvkIsAnyFlagEnabled(vkImageUsageFlags, (VK_IMAGE_USAGE_TRANSFER_SRC_BIT |	 		// May use temp view if transfer involves format change
+	if (mvkIsAnyFlagEnabled(vkImageUsageFlags, (VK_IMAGE_USAGE_TRANSFER_SRC_BIT |                   // May use temp view if transfer involves format change
 												VK_IMAGE_USAGE_SAMPLED_BIT |
 												VK_IMAGE_USAGE_STORAGE_BIT |
 												VK_IMAGE_USAGE_INPUT_ATTACHMENT_BIT |
 												VK_IMAGE_USAGE_COLOR_ATTACHMENT_BIT |
 												VK_IMAGE_USAGE_DEPTH_STENCIL_ATTACHMENT_BIT)) &&
 		(isColorFormat || (isCombinedDepthStencilFmt && supportsStencilViews))) {
 
-		mvkEnableFlags(mtlUsage, MTLTextureUsagePixelFormatView);
+		// MTLTextureUsagePixelFormatView is required only for reinterpretation of formats.  vkCmdCopyImage
+		// allows format reinterpretations for formats of the same bits, so this must be set for VK_IMAGE_USAGE_TRANSFER_SRC_BIT.
+		// However, on iOS GPU family 5 and later, this will disable lossless compression causing a performance impact,
+		// so there is an MVKConfig option to disable this (allowImageCopyReinterpretation).  It's the application's
+		// way of telling MoltenVK that it doesn't require format converting copies.  This also requires nativeTextureSwizzle
+		// in order to not need format reinterpretation for swizzling
+		if (!allowImageCopyReinterpretation || !nativeTextureSwizzle) {
+			mvkEnableFlags(mtlUsage, MTLTextureUsagePixelFormatView);
```

Comment:
Is this logic correct? From the description, it seems that if I set `allowImageCopyReinterpretation` to `false`, I am intending that I DON'T want `MTLTextureUsagePixelFormatView` enabled (unless native swizzle is missing)...but this logic seems to be doing the opposite. Am I missing something?

Reply:
Ahh, thank you I made a mistake here both on Source 2 and MVK where I inverted the meaning in my head.  Will fix.